lib/taurus/qt/qtgui/panel/taurusvalue.py

```diff
@@ -1187,8 +1190,17 @@ def applyConfig(self, configdict, **kwargs):
         for key in ('LabelWidget', 'ReadWidget', 'WriteWidget', 'UnitsWidget', 'CustomWidget', 'ExtraWidget'):
             if key in configdict:
                 widget_configdict = configdict[key]
-                getattr(self, 'set%sClass' % key)(
-                    widget_configdict.get('classid', None))
+                if ":" in widget_configdict.get('classid', None):
```

Comment:
This will fail if `widget_configdict` is `None`

Reply:
Fixed in 420e1cc